ComponentKit/TransactionalDataSources/Common/CKTransactionalComponentDataSourceAppliedChanges.h

```diff
@@ -32,4 +32,6 @@
                        insertedIndexPaths:(NSSet *)insertedIndexPaths
                                  userInfo:(NSDictionary *)userInfo NS_DESIGNATED_INITIALIZER;
 
+- (NSIndexPath *)newIndexPathForPreviousIndexPath:(NSIndexPath *)indexPath;
```

Comment:
Instead of adding this as an instance method on `CKTransactionalComponentDataSourceAppliedChanges` could this be implemented as a standalone function with its own set of tests?